test/feedhandler.cpp

```diff
@@ -0,0 +1,155 @@
+#include "3rd-party/catch.hpp"
+
+#include <memory>
+
+#include "cache.h"
+#include "configcontainer.h"
+#include "feedhandler.h"
+#include "rss.h"
+#include "test-helpers.h"
+
+using namespace newsboat;
+
+namespace {
+
+std::vector<std::shared_ptr<rss_feed>> get_five_empty_feeds()
+{
+	std::vector<std::shared_ptr<rss_feed>> feeds;
+	TestHelpers::TempFile dbfile;
+	std::unique_ptr<configcontainer> cfg(new configcontainer());
+	std::unique_ptr<cache> rsscache(new cache(dbfile.getPath(), cfg.get()));
```

Comment:
Since you don't re-open the cache file later, just use `":memory:"` here—it'll avoid costly I/O.

Reply:
Thanks!